middleware/logger/logger_test.go

```diff
@@ -181,6 +182,37 @@ func Test_Logger_ErrorTimeZone(t *testing.T) {
 	require.Equal(t, fiber.StatusNotFound, resp.StatusCode)
 }
 
+// go test -run Test_Logger_Fiber_Logger
+func Test_Logger_Fiber_Logger(t *testing.T) {
+	t.Parallel()
+	app := fiber.New()
+
+	buf := bytebufferpool.Get()
+	defer bytebufferpool.Put(buf)
+
+	customLoggerFunc := func(_ fiber.Ctx, data *Data, cfg Config) error {
+		cfg.Logger.SetOutput(cfg.Output)
+		cfg.Logger.SetFlags(0)
+		cfg.Logger.Error(data.ChainErr.Error())
+		return nil
+	}
+
+	app.Use(New(Config{
+		Output:     buf,
+		Logger:     fiberlog.DefaultLogger(),
```

Comment:
please add a second test where you inject another logger and show that it receives the data